site/docs/v1/tech/core-concepts/integration-points.adoc

```diff
@@ -0,0 +1,84 @@
+
+==== OAuth/OIDC 
+
+The first option is to use OAuth/OIDC. These are both standards and FusionAuth should work with any library or application which supports them. If you find a library which supports OAuth or OIDC and does not work with FusionAuth, please https://github.com/FusionAuth/fusionauth-issues/issues/[open a bug], as we want to know about it. You can also use a link:/docs/v1/tech/client-libraries/[FusionAuth client library].
+
+Using OAuth/OIDC lets your users authenticate and authorize; they'll receive the responses link:/docs/v1/tech/oauth/[documented for each grant]. 
+
+When you use this option, the data your client receives about the user is limited. You can put custom claims in your JWT using link:/docs/v1/tech/lambdas/[lambdas] if what you need is in the user or the registration objects. 
+
+If this level of integration meets your needs, you'll have more portability and less lock-in to FusionAuth.
+
+==== Login API
+
+You can also use the link:/docs/v1/tech/apis/login[Login API] to sign your users in directly. In this case, you will likely use one of the link:/docs/v1/tech/client-libraries/[FusionAuth client libraries].
+
+When a user signs in with this API, you receive the entire user object, as well as the JWT.
+
+When you build on top of the Login API, you're responsible for building the user interface for all login use cases. This gives you more control, as well. You can still use other FusionAuth provided user interfaces for use cases such as the forgot password flow.
+
+==== Federated Authentication
+
+Federated authentication, where FusionAuth isn't the system of record for users, is provided by link:/docs/v1/tech/apis/connectors/[Connectors] and link:/docs/v1/tech/identity-providers/[Identity Providers].
+
+Please consult Connector or Identity Provider documentation for more.
+
+=== FusionAuth APIs
+
+Whether you use the Login API or an OAuth grant, you can use additional link:/docs/v1/tech/apis/[FusionAuth APIs] in your application. 
+
+These allow your application to access and modify data not available from OIDC/OAuth. 
+
+Tasks like registering a user to an application, removing them from a group, capturing a consent, or capturing custom data all are possible via custom APIs.
+
+The upside of using these is the ability to leverage the FusionAuth data model and functionality. The downside is that your application is coupled to FusionAuth.
+
+=== Hosted Login Pages
+
+You'll see the phrase "hosted login pages" used throughout the FusionAuth site. These are all the pages that your end user sees when you are hosting your login experience on FusionAuth, as opposed to within your application. These pages are link:/docs/v1/tech/themes/[themeable].
+
+Using the hosted login pages has a number of advantages. All of the complexity of a number of different use cases is handled by FusionAuth. These include, but are not limited to, the following:
+
+* Sign in
+* Sign out
```

Comment:
Do we use this terminology elsewhere? Login and Logout would have come to mind before Sign in and Sign out.